src/reference/asciidoc/kafka.adoc

```diff
@@ -2745,7 +2745,7 @@ A `JacksonMimeTypeModule` can be registered as a bean in the application context
 Also starting with version 2.3, the `JsonDeserializer` provides `TypeReference`-based constructors for better handling of target generic container types.
 
 Starting with version 2.1, you can convey type information in record `Headers`, allowing the handling of multiple types.
-In addition, you can configure the serializer and deserializer by using the following Kafka properties:
+In addition, you can configure the serializer and deserializer by using the following Kafka properties. They have no effect if you have provided `Serializer` and `Deserializer` instances for `KafkaConsumer` and `KafkaProducer`, respectively.
```

Comment:
Sounds good!
Please, move this new sentence on a new line: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Also, please, add you name to the `@author` list of that modified `KafkaMessageListenerContainerTests`

Reply:
Thanks. Done!

Reply:
So, push the commit and we will merge your contribution!